skrub/tests/test_table_vectorizer.py

```diff
@@ -645,3 +647,148 @@ def test_mixed_types():
         "low_card_cat": [3],
     }
     check_same_transformers(expected_transformers_array, table_vec.transformers)
+
+
+def test_column_by_column():
+    X = _get_clean_dataframe()
+    table_vec_all_cols = TableVectorizer(
+        high_card_cat_transformer=GapEncoder(n_components=2, random_state=0),
+        cardinality_threshold=4,
+    )
+    table_vec_all_cols.fit(X)
+    for col in X.columns:
+        table_vec_one_col = TableVectorizer(
+            high_card_cat_transformer=GapEncoder(n_components=2, random_state=0),
+            cardinality_threshold=4,
+        )
+        table_vec_one_col.fit(X[[col]])
+        assert table_vec_one_col.get_feature_names_out() == [
+            feat
+            for feat in table_vec_all_cols.get_feature_names_out()
+            if feat.startswith(col)
+        ]
+
+
+@skip_if_no_parallel
+def test_parallelism() -> None:
+    # Test that parallelism works
+    X = _get_clean_dataframe()
+    # the gap encoder and the minhashencoder
+    # should be parallelized on all columns
+    # the one hot encoder should not be parallelized
+    for high_card_cat_transformer in [
+        lambda: GapEncoder(n_components=2, random_state=0),
+        OneHotEncoder,
+        lambda: MinHashEncoder(n_components=2),
+    ]:
+        table_vec_no_parallel = TableVectorizer(
+            high_card_cat_transformer=high_card_cat_transformer(),
+            cardinality_threshold=4,
+        )
+        X_trans = table_vec_no_parallel.fit_transform(X)
+        for joblib_backend in ["loky", "threading", "multiprocessing"]:
```

Comment:
We probably don't care about running it with each backend, I'd stick with loky, since it's essentially the same thing as multiprocessing, and threading is limited by the GIL.